tools/structure/src/main.py

```diff
@@ -61,6 +63,7 @@ def run(
 
         prompt_service_resp = None
         _, file_name = os.path.split(input_file)
+        tool_data_dir = Path(self.get_env_or_die(SettingsKeys.TOOL_DATA_DIR))
```

Comment:
NIT: You could reuse [this ](https://github.com/Zipstack/unstract-sdk/blob/29a22c682c0cd5ae090ff375db217bfcd2fa2e9f/src/unstract/sdk/tool/base.py#L96-L109) from the SDK. If you do, make sure you make it public by removing the `_` prefix